student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
```

Comment:
- [ ] According to the requirement you should add the Student class that Inherits from the Person, with the parameter `age, classroom, name = "Unknown", parent_permission = true` that is:
````ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
````